blocks/related-resources/related-resources.js

```diff
@@ -43,6 +43,35 @@ async function setRowDetails(row, block) {
   }
 }
 
+const linkTypeParsers = [
```

Comment:
`lib-franklin` already has a `getButtonIcon` function that returns icons class names based on an anchor href. https://github.com/hlxsites/merative2/blob/main/scripts/lib-franklin.js#L482

Instead of creating duplicate logic and new class names would it be possible for `related-resources` to just leverage the same logic and styling done by global buttons? What makes these buttons unique? Can't they just be another variant of global buttons instead of something that is unique to related-resources only?